File: IbolyaGergo-cs50-problems-2019-x-mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    int k;
    int l;
    int m;
    
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 ^ height > 8);
        
    for (k = 0; k < height; k++)
    {
        for (m = k+1; m < height; m++)
        {
            printf (" ");
        }
        
        for (l = 0; l < k+1; l++)
        {
            printf ("#");
            }
        
        printf ("  ");
        
        for (l = 0; l < k+1; l++)
        {
            printf ("#");
            }
 
        printf ("\n");
        }
    }
    
    

File: IbolyaGergo-cs50-problems-2022-x-sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

# Prompt the user for the pyramid's height between 1 and 8, inclusive
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break
# Creating the pyramid
for i in range(n):
    print(" " * (n - i - 1), end="#" * (i + 1))
    print(" " * 2, end="#" * (i + 1))
    print()